src/foam/dao/pg/ConnectionPool.js

```diff
@@ -0,0 +1,91 @@
+foam.CLASS({
+  package: 'foam.dao.pg',
+  name: 'ConnectionPool',
+
+  documentation: 'Represents a database connection pool',
+
+  implements: [ 'foam.nanos.NanoService' ],
+
+  javaImports: [ 'org.apache.commons.dbcp2.BasicDataSource' ],
+
+  properties: [
+    {
+      class: 'Object',
+      name: 'pool',
+      javaType: 'BasicDataSource',
+      documentation: 'Connection pool',
+    },
+    {
+      class: 'Int',
+      name: 'poolSize',
+      value: 4,
```

Comment:
4 seems too small. A few long-running queries could starve the rest of the app. I don't know what the best value would be, but maybe 20 would be better.

Reply:
Changed to 20